designs/administrator/olm/data-list-operator-hub/index.md

```diff
@@ -0,0 +1,24 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# List view on OperatorHub
+
+In this story the list view demonstrates the Operator tiles in a new light by allowing the user to view the Operators in a list view. The list view shows more operators in a list view in order to expose more information to the user in a readable way.
```

Comment:
Thanks for the review, Liz! What I was trying to say above is that the list view actually does show "more operators" because it is a condensed view. I'll try to clarify that.

Reply:
This should be updated now!